packages/core/scripts/severless-cloud-run/CloudRunHub.js

```diff
@@ -39,24 +37,29 @@ const datastore = new Datastore();
 const Web3 = require("web3");
 
 const { Logger, waitForLogger } = require("@umaprotocol/financial-templates-lib");
+let logger;
+let protocolRunnerUrl;
+let customNodeUrl;
+let hubConfig = { configRetrieval: "localStorage", saveQueriedBlock: "localStorage", spokeRunner: "localStorage" };
```

Comment:
each component of the hub can be individually configured, including a combination. I've tested retrieving remote configs and then running the hub locally with a local spoke instance for example.